### PR TITLE
auto SD logging / camera control UI

### LINKED ISSUE #1467: UI for WiFi camera control
Need setup UI for WiFi camera control

* Camera type - one ID handles camera and model
Triggering emulates auto-logger capabilities:
* Trigger speed to start recording
* Trigger speed to stop recording

### PR DESCRIPTION
**Pull request by brentpicasso**
image

Resolves #1467  #1321

### CODE REVIEW

File: autosportlabs/racecapture/config/rcpconfig.py [hunk 1]
```diff
@@ -23,6 +23,7 @@
 from autosportlabs.racecapture.geo.geopoint import GeoPoint
 from kivy.logger import Logger
 from distutils.version import StrictVersion
+from _ast import Or
```
Comment (ddimensia) [inline autosportlabs/racecapture/config/rcpconfig.py]: Not seeing where this is used, a little surprising your pulling in the ast

File: autosportlabs/racecapture/views/configuration/rcp/configview.py [hunk 2]
```diff
@@ -220,6 +221,8 @@ def create_scripting_view(capabilities):
 
         attach_node('OBDII', None, lambda: OBD2ChannelsView(channels=runtime_channels, base_dir=self.base_dir))
 
+        attach_node('Automatic', None, lambda: AutoControlConfigView(channels=runtime_channels))
```
Comment (ddimensia) [inline autosportlabs/racecapture/views/configuration/rcp/configview.py]: I'm not sure if there's a better name but this is not very self explanatory in the ui. I'd almost prefer something like 'Automate' although even that send a bit off to me. It's at least closer to the intent of the settings in this view.

Comment (brentpicasso) [reply autosportlabs/racecapture/views/configuration/rcp/configview.py]: I think 'Automatic Control' should work.  I meant to update this before pushing. Thanks.

Comment (brentpicasso) [reply autosportlabs/racecapture/views/configuration/rcp/configview.py]: fixed.

**Comment by ddimensia** [comment]
Not sure if this was merged or not but the changes LGTM.

**Comment by brentpicasso** [comment]
Yup, changes were merged. Thanks for the review!

**Pull request merged**